clang/test/Driver/cheri/freebsd-cheri.c

```diff
@@ -1,52 +1,42 @@
 // RUN: %plain_clang_cheri_triple_allowed -no-canonical-prefixes \
 // RUN:   -target cheri-pc-freebsd11 %s                              \
```

Comment:
Since your changing most of this file anyway could you also remove the "cheri-" triples?